ts/features/idpay/initiative/configuration/screens/PaymentMethodChoiceScreen.tsx

```diff
@@ -0,0 +1,151 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { List, ListItem, Text, View } from "native-base";
+import React, { useRef } from "react";
+import { StyleSheet } from "react-native";
+import { Wallet } from "../../../../../../definitions/pagopa/Wallet";
+import { Body } from "../../../../../components/core/typography/Body";
+import { H1 } from "../../../../../components/core/typography/H1";
+import { H4 } from "../../../../../components/core/typography/H4";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import BaseScreenComponent from "../../../../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../../../../components/ui/FooterWithButtons";
+import Switch from "../../../../../components/ui/Switch";
+import TypedI18n from "../../../../../i18n";
```

Comment:
In other screen of the app the import should be
`import I18n from "../../../../../i18n";`
What do you think about it?